src/ripple/app/tx/impl/SetAccount.cpp

```diff
@@ -218,6 +218,37 @@ SetAccount::preclaim(PreclaimContext const& ctx)
         }
     }
 
+    //
+    // Clawback
+    //
+    if (ctx.view.rules().enabled(featureClawback))
+    {
+        if (uSetFlag == asfAllowClawback)
+        {
+            if (uFlagsIn & lsfNoFreeze)
+            {
+                JLOG(ctx.j.trace()) << "Can't set Clawback if NoFreeze is set";
+                return tecNO_PERMISSION;
+            }
+
+            if (!dirIsEmpty(ctx.view, keylet::ownerDir(id)))
```

Comment:
How is `Clawback` triggering this piece of code which is in `SetAccount` ? I grep'ped for `SetAccount` in `Clawback.cpp` but I couldn't find any instances